newIDE/app/src/UI/Search/CategoryChooser.js

```diff
@@ -68,12 +70,15 @@ const MemoizedTagsTree = React.memo<MemoizedTagsTreeProps>(function TagsTree({
           : ''
       }
       defaultExpanded={[]}
-      onNodeSelect={() => {}}
+      onNodeSelect={() => setIsOnHomePage && setIsOnHomePage(false)}
```

Comment:
A way to see something is wrong: setIsOnHomePage is called always with false.
This means the prop is not fully used and so is not representing what it's really doing.
It seems here it's just calling it when a label is chosen?

Reply:
Yes, I though about it too and kept it like this because it will be removed in a few day.
Do you think it must be done?

Reply:
Well I'm always worried of "this is just temporary and I will remove it for sure in two days" 😇
But if you think that's a detail, then no big deal.

Reply:
No, you're right it might take some time before the old filters can be removed. I made the change.